client/src/components/DropdownMenu/index.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import { Menu } from './styled'
+import { Menu, NavLink } from './styled'
 import { SmallText, Divider } from '../../styles/Text'
-import { logout } from '../../auth'
+import { logout, useAuth } from '../../auth'
 
-const DropdownMenu = ({ isOpen }) => {
+const DropdownMenu = ({ isOpen, background, color }) => {
+  const [logged] = useAuth()
+  if (!logged) {
+    return (
+      <Menu background={background} color={color} isOpen={isOpen}>
+        <SmallText>
+          <NavLink to="/login">Log In</NavLink>
+        </SmallText>
+        <Divider />
+        <SmallText>
+          <NavLink to="/signup">Sign Up</NavLink>
+        </SmallText>
+      </Menu>
+    )
+  }
```

Comment:
Minor: not a big thing, but it's good practice to add new lines to separate chunks of logic where it makes sense. One good place to put a new line is return statements.